configure.ac

```diff
@@ -382,13 +382,17 @@ dnl User setting: Debug mode (off by default)
 dnl
 AC_ARG_ENABLE([debug],
     [AS_HELP_STRING([--enable-debug], [enable debug mode])],
-    [AC_DEFINE([GAP_KERNEL_DEBUG], [1], [define if building in debug mode])
-     AC_DEFINE([GAP_PRINT_BACKTRACE], [1], [to enable backtraces upon crashes])],
+    []
     [enable_debug=no]
```

Comment:
The fact that all these features are disabled by default suggest yet another way to fix this: drop the 2nd & 3rd argument) (i.e. also the `enable_debug=no` etc.), and then use the ` ... = "xyes"` instead of `... != "xno"` (see also [Example 1.1 in here](https://autotools.io/autoconf/arguments.html)). This is arguably the idiomatic approach for doing it.